doc/new-guidelines.md

```diff
@@ -687,6 +687,17 @@ actually subscribing to them, so reframe's signal graph gets validated too.
     (is (= expected (rf/sub [sub-name])))))
 ```
 
+#### Component tests
```

Comment:
perhaps a better location for this is in the quo directory - there is a readMe there? 
https://github.com/status-im/status-mobile/blob/develop/src/quo/README.md

Reply:
@J-Son89, not sure about this. I think that document tree in the main readme file is more convenient, because it is likely to be read. The chance that people will find this piece of information in some other directory is much lower. For example I didn't even know that `quo` directory has a readme... just never noticed

Reply:
To me both places seem okay to have this information, 
I'd lean more towards Quo readme because this guideline specifically is related to quo component tests and would be good to maybe have that linked somehow here in guidelines document.
No strong opinions however :)

Reply:
I think dispersing the READMEs in top-level directories is a common practice, but easy to overlook. Good point @vkjr, maybe we should also have a top-level table of contents on the main guidelines file pointing to secondary READMEs.

Reply:
Moved the text about no-props test to quo/readme, added link to that readme in guidelines, under "project structure" section, wdyt?